website/src/pages/_app.tsx

```diff
@@ -0,0 +1,5 @@
+import '@theguild/components/style.css';
+
+export default function App({Component, pageProps}: any) {
```

Comment:
```suggestion
export default function App({Component, pageProps}: AppProps) {
```
Import from next/app